cpu_tests/test_streaming_iterators.py

```diff
@@ -40,11 +41,41 @@ def get_simple_dataset():
             torch.LongTensor([8, 9]),
         ]
     )
-    dataset = DeferredDataset(dataset)
-    dataset = SkipDeferredDataset(dataset, 0)
+    dataset = DocumentToSequenceDataset(
+        dataset,
+        block_size=None,
+        permute_documents=False,
+        break_mode="passthrough",
```

Comment:
This is a new breakmode. What is it? 😅

Reply:
A few of tests for saving/restoring state of iterators do not actually form sequences and just test loading single documents. Since I collapsed the skipping behavior into the SequencesToDocument object, I added "passthrough" to skip the construction of sequences and just return a whole document so those tests still do the same thing.